src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsTerminal/Helpers/TerminalHelper.cs

```diff
@@ -15,9 +15,20 @@ public static class TerminalHelper
         /// </summary>
         /// <param name="profileName">Name of the Terminal profile</param>
         /// <param name="openNewTab">Whether to launch the profile in a new tab</param>
-        public static string GetArguments(string profileName, bool openNewTab)
+        /// <param name="openQuake">Whether to launch the profile in the quake window</param>
+        public static string GetArguments(string profileName, bool openNewTab, bool openQuake)
         {
-            return openNewTab ? $"--window 0 nt --profile \"{profileName}\"" : $"--profile \"{profileName}\"";
+            var argsPrefix = string.Empty;
+            if (openQuake)
+            {
+                argsPrefix = "--window _quake nt";
+            }
+            else if (openNewTab)
+            {
+                argsPrefix = "--window 0 nt";
+            }
```

Comment:
if both is true we should at least log an error and maybe show an error instead opening wt. The normal user don't knows what has priority and might be confused if enable NewTab doesn't work.

Reply:
I tested it and of course quake window win.
I think a warning here may bloat the log file and we don't want it.

Reply:
In a way, the quake window could be considered a special case of opening a new tab or new window. I don't think it would be too counterintuitive, but we could add a short comment in the description of the quake setting?

Reply:
Hmm. Does it open a new tab and shows it in quake mode or does it open a new window and shows it in quake mode?

Reply:
It opens in the quake window. Quake window doesn't have tabs.

Reply:
The the `nt` in the parameter makes no sense. Correct?

Reply:
@FWest98 
A short note in the description of quake window setting makes sense as in this mode you can't open a second tab/window when having the parameter.

Reply:
There is only one unique quake window. It does have tabs, but they are hidden by default ("focus mode").

Reply:
did a small test:
1. It is possible to have one quake window as admin and one in normal mode running. Then you can switch between them in your tray area.
2. If you open a second tab/quake window using the same pernicions it's very hard to switch between them. So opening tabs in quake mode might be an edge case.

Reply:
1. That is true
2. When you have the option to open in quake, there is no difference between opening a new tab or not. They always open a new tab in quake (or a new window when there is no quake window yet). You can show the tab bar using Ctrl+Shift+P and typing "focus mode" and toggling it. Then it behaves like a normal window.

Reply:
Then I think if user know how quake mode works (what we can expect here), a hint in the description isn't required.

Reply:
@FWest98 with "opening a new tab in quake" do you mean opening more than 1 quake window?
According to documentation https://docs.microsoft.com/en-us/windows/terminal/tips-and-tricks#quake-mode `nt` can be removed from the command for open the quake window.

Reply:
@davidegiacometti 
It opens a real new tab in the existing quake window. With and without `nt` param.

Reply:
Talk about tabs and quake doesn't make any sense to me.
Disable focus mode to quake window it's also strange 😅

Reply:
Quake mode is a bit of an odd one out still but at least for me (and many others) a very much welcome feature :P In the long term the WT team is planning to change and streamline the implementation, but until then I think this is the best we can do. I will update the PR!

Reply:
In the setting description, I clarified that this option always opens a new tab in the quake window. In the code for this logic, I added a comment describing the WT behaviour. I hope this is sufficiently clear now!